archivist/confirmer.py

```diff
@@ -68,8 +95,8 @@ def _wait_for_confirmation(self, identity):
     if entity[CONFIRMATION_STATUS] == CONFIRMATION_CONFIRMED:
         return entity
 
-    return None  # type: ignore
-    # pylance does not correctly interoperate the decorator usage, and thinks that
+    return None
+    # pylance does not correctly interprate decorator usage, and thinks that
```

Comment:
ah - interpret

Reply:
I think with the overloads we don't need that comment any more